htlcswitch/interfaces.go

```diff
@@ -100,22 +100,21 @@ type ChannelLink interface {
 	// policy to govern if it an incoming HTLC should be forwarded or not.
 	UpdateForwardingPolicy(ForwardingPolicy)
 
-	// HtlcSatifiesPolicy should return a nil error if the passed HTLC
-	// details satisfy the current forwarding policy fo the target link.
-	// Otherwise, a valid protocol failure message should be returned in
-	// order to signal to the source of the HTLC, the policy consistency
-	// issue.
-	HtlcSatifiesPolicy(payHash [32]byte, incomingAmt lnwire.MilliSatoshi,
+	// CanForwardHtlc should return a nil error if the passed HTLC details
+	// satisfy the current forwarding policy fo the target link. Otherwise,
+	// a valid protocol failure message should be returned in order to
+	// signal to the source of the HTLC, the policy consistency issue.
+	CanForwardHtlc(payHash [32]byte, incomingAmt lnwire.MilliSatoshi,
```

Comment:
I don't think these new names really fit, one would expect this method to return a _bool_ based on its name.

Reply:
Good point. I made this one `CheckHtlcForward`